napari/components/_tests/test_layer_slicer.py

```diff
@@ -54,6 +54,12 @@ def _update_slice_response(self, response: FakeSliceResponse):
     def _slice_dims(self, *args, **kwargs) -> None:
         self.slice_count += 1
 
+    def _set_loaded(self, loaded: bool) -> None:
+        self._loaded = loaded
+
+    def _set_unloaded_slice_id(self, slice_id: int) -> None:
+        self._last_slice_id = slice_id
+
```

Comment:
Why add these here if we aren't going to use them in testing? I have mixed feelings on this - on one hand I appreciate the Fake Layer being more like a real layer, but on the other, I'd feel more comfortable testing these properties on real code.

Reply:
I removed `_set_loaded` for now since that is not needed. We also discussed using real layers now to avoid these fakes, which could be doable either as part of a another separate PR like https://github.com/andy-sweet/napari/pull/51 or in a follow-up PR.